Edits to Equivariance chapter

ElanaPearl opened the pull request:
* fixed a few random typos
* changed the order of rotations in the finite group example 13.3.1 since the text and images state that we rotate twice, then five times but the written equation is r^2 . r^5 (which would imply rotating five times _then_ twice)
* the multiplication table for p4m had a few values that I think were incorrect so I fixed those (see before / after tables)
    * <img width="440" alt="before" src="https://user-images.githubusercontent.com/12026888/132432822-dc8b468c-91bf-44bf-9179-dae8998ec54e.png">
    * <img width="433" alt="after" src="https://user-images.githubusercontent.com/12026888/132432850-d90cdaf9-483b-4971-aa29-e7466cd9d84e.png">
* the example showing that the p4m group is nonabelian had some small errors in the images and the equations so fixed those, see image below for more info + logic
    * in the original it states that r.s = rs and s.r = r^3s but I think it should be the other way around such that r.s = r^3s and s.r = rs
    * the images for r.s do an s operation followed by a r^2s operation (instead of followed by an r operation), I think because the image is quad-1-1 but that does 1 rotation _then_ 1 mirror
    * the images for s . r do an r operation followed by an r^2s operation (instead of of an s operation)
    * I tried to fix them to now look like this and hopefully this diagram explains why I think the new equations are correct
    * <img width="622" alt="explanation_of_p4m_lack_of_commutativity" src="https://user-images.githubusercontent.com/12026888/132433087-5f7b0d38-0947-4c3c-a9f2-345c7009978d.png">

small caveat/note: I'm very new to group theory so there's a chance I made a mistake in some part of this but I _think_ it's all correct

ElanaPearl commented:
switched to draft PR for now as I realized my editor reformatted many miscellaneous part of the notebook making this PR overly complicated looking

whitead commented:
@ElanaPearl Oof, this is tricky stuff! I agree on the table and 13.31. Thanks for catching that and taking the time to fix it! This is so helpful. 

For the p4m [Cayley table](https://en.wikipedia.org/wiki/Cayley_table), r . s => row r, col s => rs, so mirror first, rotate second. I think what is written is correct then? You're right that on the s . r images the second one is wrong. I should probably define and discuss convention on Cayley table here, since it's not obvious. Does that make sense?

ElanaPearl commented:
Ah yes that makes sense, so the text was initially correct. I think I misinterpreted the order of operations in the Cayley table and assumed (row i, col j) mean you do operation i first then operation j (aka j . i) in which was incorrect. Thanks for explaining, I just switched the text back.

I still think both of the images above showing the `r . s` and `s . r` operations I adjusted should still be correct though so I left those changes in.
* The image you get from calling `quad-1-1` isn't actually the right image for `r . s = rs` since the quad function does the rotations first, so it does 1 rotation followed by 1 mirror, which actually happens to be exactly what you want for  `s . r`
* So the image for `s . r` should be `quad-1-1`, and the image for `r . s` should be something else
* For `r . s` the image should be `quad-3-1` since doing 3 rotations <img width="57" alt="3 rotations" src="https://user-images.githubusercontent.com/12026888/132541230-028b5003-3de3-4f78-a8fe-c7c3a2776f92.png"> followed by 1 mirror <img width="57" alt="3 rot 1 mirror" src="https://user-images.githubusercontent.com/12026888/132540869-90fc999d-818e-4f84-9fa3-e1f029aa30fa.png"> is equivalent to 1 mirror <img width="55" alt="1 mirror " src="https://user-images.githubusercontent.com/12026888/132541353-b22912dc-b60d-4478-bea5-d53e8e8977e5.png"> followed by 1 rotation <img width="57" alt="1 mirror 1 rot" src="https://user-images.githubusercontent.com/12026888/132540926-3197b876-ae89-4b6c-8746-4cbc8a49619a.png">
* Also wow keeping track of all of these is can get confusing

And yes it would definitely be helpful to add a brief explanation! Even something as simple as just mentioning the phrase Cayley table would be helpful so people like myself who'd never seen them can know what term to look up if they want to understand the table better.

whitead commented:
Great, thanks for the quick update! I'll add a section on the Cayley table and link to wikie.
